### PR TITLE
Bug fix for phone link when duplicated prefix 'tel'

### PR DESCRIPTION
**Pull request by ShadowTourist**
Bug fix for phone link when duplicated prefix 'tel'

### CODE REVIEW

**Comment by justinguo** [comment]
@ShadowTourist This PR looks good to me. Can you check if the CLA has been signed? You might click on the above link or click the recheck link if you already signed. Ideally, after CLA is signed, then all PR jobs in CI process will run for each PR.

**Comment by ShadowTourist** [comment]
> @ShadowTourist This PR looks good to me. Can you check if the CLA has been signed? You might click on the above link or click the recheck link if you already signed. Ideally, after CLA is signed, then all PR jobs in CI process will run for each PR.

CLA signed now.

**Comment by MarcoEidinger** [comment]
Ignore failing beta build. In future please use https://github.com/streamich/git-cz to ensure proper commit message as CHANGELOG will be generated based on commit message information. No action needed here as I will squash and merge with new commit message.

**Comment by justinguo** [comment]
@ShadowTourist Can you check `streamich/git-cz` out to see if you can formate the commit message correctly? If you like you can create another commit with formatted message. As a reference, a good name for this commit can be:
`fix: 🐛 fix phone link in button for duplicated tel prefix`. As long as the commit message is clear and specific then it would be fine.

File: Sources/SAPCAI/UI/Common/SwiftUI/ButtonView.swift [hunk 1]
```diff
@@ -50,6 +50,14 @@ struct ButtonView: View {
             }
         })
     }
+    
+    func optimizePrefix(for phoneNumber: String) -> String {
+        if phoneNumber.lowercased().hasPrefix("tel:") {
+            return phoneNumber
```
Comment (MarcoEidinger) [inline Sources/SAPCAI/UI/Common/SwiftUI/ButtonView.swift:56]:
```suggestion
            return phoneNumber.filter { !$0.isWhitespace }
```

**Review by MarcoEidinger — APPROVED** [review]
lgtm 👍 

P.S.: I changed my mind last minute to do additional formatting (i.e. removing whitespace) to make the example (DemoBot: "Demo 6" utterance) work

**Comment by MarcoEidinger** [comment]
@justinguo I will squash and merge with new commit message. 

@ShadowTourist for the future use tools(like git-cz) to ensure conventional commits

**Pull request merged**